advanced_alchemy/config/sync.py

```diff
@@ -51,14 +56,27 @@ class SQLAlchemySyncConfig(GenericSQLAlchemyConfig[Engine, Session, sessionmaker
     """
 
     def __post_init__(self) -> None:
+        """Initialize the configuration after dataclass initialization.
+
+        Sets the metadata on the alembic config if provided.
+        """
         if self.metadata:
             self.alembic_config.target_metadata = self.metadata
         super().__post_init__()
 
     @contextmanager
-    def get_session(
-        self,
-    ) -> Generator[Session, None, None]:
+    def get_session(self) -> Generator[Session, None, None]:
+        """Get a session context manager.
+
+        Returns:
```

Comment:
Should this be yields?
```suggestion
        Yields:
```